en/identity-server/7.0.0/docs/assets/code-samples/body.html

```diff
@@ -0,0 +1,84 @@
+    <div class="fp-header-logo">
+        {{{ProductHeader}}}
+    </div>
+    <div class="fp-header-theme">
+        <input type="checkbox" class="fp-checkbox" id="fp-checkbox">
+        <label for="fp-checkbox" class="fp-theme-checkbox-label">
+            <i class="moon icon"></i>
+            <i class="sun icon"></i>
+            <div class="fp-theme-indicator"></div>
+        </label>
+    </div>
+</header>
+<main class="center-segment">
+    <div class="ui segment fp-container">
+        <div class="fp-info-container">
+            <div class="fp-info-illustration">
+                <img src="extensions/layouts/custom/assets/illustration.svg" alt="Auth Page Illustration Image">
+            </div>
+            <div class="fp-info-content">
+                <p>Access Our All Service With This Account</p>
+                <div class="fp-services">
+                    <div class="fp-service-item">
+                        <i class="utensils icon"></i>
+                        <p>Create new orders</p>
+                    </div>
+                    <div class="fp-service-item">
+                        <i class="shipping fast icon"></i>
+                        <p>Get delivery to doorstep</p>
+                    </div>
+                    <div class="fp-service-item">
+                        <i class="credit card icon"></i>
+                        <p>Online payment facility</p>
+                    </div>
+                    <div class="fp-service-item">
+                        <i class="money bill alternate outline icon"></i>
+                        <p>Cash on delivery</p>
+                    </div>
+                    <div class="fp-service-item">
+                        <i class="map marker alternate icon"></i>
+                        <p>Track your order</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="fp-divider"></div>
+        <div class="fp-content-container">
+            <div class="center-segment">
+                <div class="ui container">
+                    {{{MainSection}}}
+                </div>
+            </div>
+        </div>
+    </div>
+</main>
+{{{ProductFooter}}}
+{{/isPolicyPage}}
+
+{{#isPolicyPage}}
+<header class="fp-header">
+    <div class="fp-header-logo">
+        {{{ProductHeader}}}
+    </div>
+    <div class="fp-header-theme">
+        <input type="checkbox" class="fp-checkbox" id="fp-checkbox">
+        <label for="fp-checkbox" class="fp-theme-checkbox-label">
+            <i class="moon icon"></i>
+            <i class="sun icon"></i>
+            <div class="fp-theme-indicator"></div>
+        </label>
+    </div>
+</header>
+<main class="policy-page">
+    <div class="app-content policy-page-content">
+        <div class="ui container">
+            {{{MainSection}}}
+        </div>
+    </div>
+</main>
+{{{ProductFooter}}}
+{{/isPolicyPage}}
+
+<script src="extensions/layouts/custom/script.js"></script>
```

Comment:
Add newline

Reply:
fixed with https://github.com/wso2/docs-is/pull/5157/commits/34741ff481d9b2fe5a4063b7e8a96a63ee6b5d1b